components/shotAccordion.js

```diff
@@ -38,13 +39,20 @@ function DataField(field, value) {
               width: 200,
             }}
           >
-            <Row name={"(Actual)"} value={numTrunc(value["carry"])} />
-            <Row name={"(Target)"} value={numTrunc(value["target"])} />
-            <Row name={"(Diff)"} value={numTrunc(value["carryDiff"])} />
+            <Row name={"(Actual)"} value={`${numTrunc(value["carry"])} yd`} />
+            <Row name={"(Target)"} value={`${numTrunc(value["target"])} yd`} />
+            <Row name={"(Diff)"} value={`${numTrunc(value["carryDiff"])} yd`} />
           </View>
         </View>
       );
+    case "target":
+      return (
+        <Row key={field} name={title[field]} value={`${numTrunc(value)} yd`} />
```

Comment:
possibly have a unit table in a constant file or on firebase

Reply:
In theory, Firebase would be better cuz it's easier to add new drills and new fields